frontend/src/pages/CompareV2.tsx

```diff
@@ -73,6 +94,23 @@ enum MetricsType {
   MARKDOWN,
 }
 
+const metricsTypeToString = (metricsType: MetricsType): string => {
```

Comment:
I created this function rather than simply assigning these values directly on the `enum` because the `MD2Tabs` component requires type `number` for its parameters like `selectedTab`.